greymatter/modules/fabric/services.tf

```diff
@@ -6,30 +45,31 @@ resource "aws_ecs_service" "control-api" {
     name = "control-api"
```

Comment:
We should concatenate control-api with another name we pass in, so that this module could be invoked multiple times without collisions.

Reply:
@dontlaugh how important do you think this change is? It can be done but it requires a little bit of thought because ill have to create a variable such as "release_name", add it to the ecs service names, dns names for connection to control-api - dns name to connect to control in the sidecar (so ill have to pass it into the sidecar as well) and probably also add it to the sidecar names. I also may need to add it to security group names, etc because tf will complain if say one module is already up and it tries to create security group with existing name.

 I can work through this now or file an issue for later creating a setup where this stuff can be invoked multiple times.

Reply:
File an issue. Won't block this.

Reply:
Issue filed https://github.com/greymatter-io/terraform-greymatter-ecs/issues/22